.github/workflows/workflow.yml

```diff
@@ -87,6 +92,17 @@ jobs:
             export PATH="/usr/local/opt/autoconf/bin:/usr/local/opt/automake/bin:/usr/local/opt/libtool/bin:$PATH"
           fi
           cd modules/bitcoin && make
+      
+      - name: Build - Libbitcoin
+        timeout-minutes: 40
+        run: |
+          if [ "${{ matrix.os }}" == "ubuntu-24.04" ]; then
+            export BOOST_LIB_DIR=/usr/lib/x86_64-linux-gnu/
+          elif [ "${{ matrix.os }}" == "macos-14" ]; then
+            export BOOST_LIB_DIR=$(brew --prefix boost@1.85)/lib/
```

Comment:
You're exporting `BOOST_LIB_DIR` and `BOOST_INCLUDE_DIR` but where are you using them?